lib/beaker/hypervisor/docker.rb

```diff
@@ -88,7 +88,12 @@ def provision
           unless host['mount_folders'].nil?
             container_opts['HostConfig'] ||= {}
             container_opts['HostConfig']['Binds'] = host['mount_folders'].values.map do |mount|
-              a = [ File.expand_path(mount['host_path']), mount['container_path'] ]
+              host_path = File.expand_path(mount['host_path'])
+              # When using docker_toolbox and getting a "(Driveletter):/" path, convert windows path to VM mount
+              if ENV['DOCKER_TOOLBOX_INSTALL_PATH'] && host_path =~ /^.\:\//
```

Comment:
Haven't used Docker Toolbox, is `DOCKER_TOOLBOX_INSTALL_PATH` a standard environment variable set up by the installer?

Reply:
Yeps, installer sets that https://github.com/docker/toolbox/blob/c971a9cb9b35f5bea1d3724702423135b530a226/windows/Toolbox.iss#L91

Its also used when running the "docker shell" executable - so should stay arround https://github.com/docker/toolbox/blob/master/windows/start.sh

---

With the path check `/^.\:\//` I belive its fairly safe to use for anyone... 
-> mac has docker_toolbox too

Only acts on `C:/` not `C:\` so when using msys/gitbash
-> should not affect cmd/powershell (`windows-on-windows-dev`?)

Having "native and toolbox" installed breaks a bunch of other things, 
->so likely "should not be our problem"